spec/puppet-server-lib/puppet/jvm/execution_spec.rb

```diff
@@ -56,6 +56,30 @@ def test_execute(command, options = {})
       expect(result.exitstatus).not_to eq 0
     end
 
+    context "it should support other shellisms" do
+      it "should support pipes" do
+        result = test_execute("echo foo | wc -c")
+        expect(result).to be_a Puppet::Util::Execution::ProcessOutput
+        expect(result.lstrip).to eq "4\n"
```

Comment:
Left strip and then expecting a result with a trailing newline seems weird to me. Why not just strip both? Or not strip either side? Is the preceding whitespace variable but the newline both mandatory and important?

Reply:
stripping it all seems fine. i didn't want to figure out how wc deals with leading whitespace.

Reply:
updated.